mender/templates/secret-artifacts-storage.yaml

```
@@ -11,14 +11,26 @@ metadata:
     app.kubernetes.io/part-of: mender
     helm.sh/chart: "{{ .Chart.Name }}"
 data:
+{{- if eq .Values.global.storage "aws" }}
   AWS_URI: {{ .Values.global.s3.AWS_URI | default .Values.global.url | default (ternary (printf "https://%s" .Values.api_gateway.service.name ) (printf "http://%s" .Values.api_gateway.service.name) (.Values.api_gateway.env.SSL)) | b64enc }}
-{{- if .Values.global.s3.AWS_EXTERNAL_URI }}
+  DEPLOYMENTS_STORAGE_BUCKET: {{ .Values.global.s3.AWS_BUCKET | b64enc }}
+  {{- if .Values.global.s3.AWS_EXTERNAL_URI }}
   AWS_EXTERNAL_URI: {{ .Values.global.s3.AWS_EXTERNAL_URI | b64enc }}
-{{- end }}
-  AWS_BUCKET: {{ .Values.global.s3.AWS_BUCKET | b64enc }}
+  {{- end }}
   AWS_REGION: {{ .Values.global.s3.AWS_REGION | b64enc }}
-{{- if and (not .Values.global.s3.AWS_SERVICE_ACCOUNT_NAME) .Values.global.s3.AWS_ACCESS_KEY_ID }}
+  {{- if and (not .Values.global.s3.AWS_SERVICE_ACCOUNT_NAME) .Values.global.s3.AWS_ACCESS_KEY_ID }}
   AWS_AUTH_KEY: {{ .Values.global.s3.AWS_ACCESS_KEY_ID | b64enc }}
   AWS_AUTH_SECRET: {{ .Values.global.s3.AWS_SECRET_ACCESS_KEY | b64enc }}
-{{- end }}
+  {{- end }}
   AWS_FORCE_PATH_STYLE: {{ .Values.global.s3.AWS_FORCE_PATH_STYLE | b64enc }}
+{{- else }}
+  DEPLOYMENTS_STORAGE_BUCKET: {{ .Values.global.azure.CONTAINER_NAME | b64enc }}
```

NOTE: This secret is included with the environment prefix set.
```suggestion
  STORAGE_BUCKET: {{ .Values.global.azure.CONTAINER_NAME | b64enc }}
```

---

Done!